Woof/Woof/View/OwnerMainTabView.swift

```diff
@@ -0,0 +1,72 @@
+import SwiftUI
+
+struct OwnerMainTabView: View {
+    @State private var selection: Tab = .sitters
+
+    init() {
+        customizeTabBar()
+    }
+
+    var body: some View {
+        NavigationView {
+            TabView(selection: $selection) {
+                Group {
+                    Text("List of pet sitters")
+                        .tabItem {
+                            Label("Sitters", systemImage: "magnifyingglass")
+                        }
+                        .tag(Tab.sitters)
+
+                    Text("Walkings stub")
+                        .tabItem {
+                            Label("Walkings", systemImage: "pawprint")
```

Comment:
Selected colour isn't displayed for two system images:
- pawprint;
- person.text.rectangle

<img width="350" alt="Screenshot 2023-05-04 at 9 33 23 PM" src="https://user-images.githubusercontent.com/120586844/236220737-fb7867e8-23bc-416f-a27b-178671546d8c.png">

Reply:
Dark purple color is for selected tab, and dark gray is for unselected tab. When we tap on the icon, its color should change from dark gray to dark purple. Does it work for you? 
Or could you please describe in more detail what you expect and what is going wrong?

Reply:
Yes, sure, the color should change from dark gray to dark purple, when we tap on the icon. My bad.